framework/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Yii Framework 2 Change Log
 - Bug #11040: Check parameter 'recursive' and disable recursive copying with option 'recursive' => false in method BaseFileHelper::copyDirectory (Ni-san)
 - Chg: HTMLPurifier dependency updated to `~4.6` (samdark)
 - Chg #11071: `yii\helpers\BaseArrayHelper::isIn()` and `isTraversable()` since now throw `\yii\base\InvalidParamException` instead of `\InvalidArgumentException` (nukkumatti)
+- Bug #11125: Fixed JSON_ERROR_SYNTAX for json_decode(null) in PHP 7
```

Comment:
Not in https://github.com/yiisoft/yii2/commit/258175149d1dceb44c382ea34339dafff3eed107